### DISCUSSION TITLE
Hey Guys I need some help

### DISCUSSION THREAD

**Discussion by Psyop100** [community/community#144273] [Discussions]
### Select Topic Area

Question

### Body

Sorry if this is a stupid question but I’m a bit of a noob! So I’m having trouble logging into my Bitstamp account. I need to reset the 2fa. Only issue is I misplaced the ID that I need to recover it. Is there any way around this? Thank you

**Comment by UplandJacob** [comment]
Try and email support@bitstamp.net. You may need to give some info to prove that it’s you and hopefully they can reset it.

**Comment by mecodeatlas** [comment]
Hi @Psyop100,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Bitstamp Support](https://www.bitstamp.net/customer-support/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Psyop100** [reply]
Thanks for the welcome and the response. I got it resolved. Thanks
